setup.json

```diff
@@ -33,7 +34,8 @@
             "pytest>=4.4,<5.0.0",
             "pytest-cov>=2.6.1,<3.0.0",
             "coverage",
-            "aiida-gaussian-datatypes"
+            "aiida-gaussian-datatypes",
+            "cp2k-output-tools"
```

Comment:
To be honest, I think those two guys should go into `install_requires`. There is no point in keeping them as test extras - they are not for tests.

Reply:
Up to this point they were complete optional (there are `import` checks in the given code parts giving a corresponding error message). They are in the test extras not because they'd be test dependencies, but to make sure the tests depending on those are being run (otherwise they'll be skipped).
I am happily making those "hard"-dependencies!

Reply:
> I am happily making those "hard"-dependencies!

Do you want to do it in a separate PR? If yes, then this one can be merged already.

Reply:
sure, because then I can also drop a couple of `try/excepts`. Thanks!